src/app/components/common/layouts/admin-layout/admin-layout.template.html

```diff
@@ -16,7 +16,7 @@
             </ng-template>
           </div>
           <ng-template #lightLogo>
-            <div class="branding mat-bg-primary" *ngIf="is_freenas; else isTruenasLight">
+            <div class="branding mat-bg-primary" *ngIf="product_type == 'CORE'; else isTruenasLight">
```

Comment:
This shouldn't be necessary, it should just always show TrueNAS Logos (the branding of TrueNAS will be same between Core and Enterprise)